R/cascade.r

```diff
@@ -52,6 +52,7 @@ cascade.grouped_svy <- function(.data, ..., .dots, .fill = NA) {
   dots <- rlang::quos(...)
 
   groups <- as.character(groups(.data))
```

Comment:
Hm, I think I should be using `group_vars()` here instead of forcing `groups` to be character.

Can you see if `groups <- group_vars(.data)` fixes this so you don't even need the gsub line?